adding RPC method GetChainID to the access protobuf to retreive chain ID (#16)
onflow/flow: docs/access-api-spec.md

  ```
</details>

---

### Chain ID

> ChainID helps identify the Flow network (mainnet versus testnet). The following method can be used to query for the Chain ID

Author: do you think it add values if we also report `uptime`? I could extend the GetChainID method to report more details about the network.

Reviewer: Yeah, I was thinking it might be worthwhile to generalize this a bit. 

Maybe `GetNetworkParameters`?

`chain_id` is just one of the parameters IMO. We might want to return information about the epoch, gas prices, etc in the future.

```
message GetNetworkParametersResponse {
    string chain_id = 1;
}
```

Author: how about `GetChainDetails`?

| transaction_index | Zero-based index of the transaction within the block |
| event_index       | Zero-based index of the event within the transaction |
| payload           | Event fields encoded as [JSON-Cadence values](/docs/json-cadence-spec.md)|

### Network Parameters

> Network parameters provide information about the flow network. Currently, it only includes chain-id

Reviewer: ```suggestion
Network parameters provide information about the Flow network. Currently, it only includes chain ID.
```

```
message NetworkParameters {
	string chain_id = 1;
}
```

| Field             | Description    |
| ------------------|----------------|

> | chain_id          | ChainID helps identify the Flow network. It can be one of `flow-mainnet`, `flow-testnet` or `flow-emulator` |

Reviewer: ```suggestion
| chain_id          | Chain ID identifies the Flow network and can be one of `flow-mainnet`, `flow-testnet` or `flow-emulator` |
```
